tests/test_build.py

```diff
@@ -38,7 +38,20 @@ def test_build(test_site_dir):
     assert opensdg_output.validate()
     assert opensdg_output.execute()
 
-    exp_dirs = set(['comb', 'data', 'edges', 'headline', 'meta', 'stats', 'zip', 'translations'])
+    # Also generate GeoJSON output.
+    geojson_output = sdg.outputs.OutputGeoJson(
+        inputs=inputs,
+        schema=schema,
+        output_folder=site_dir,
+        translations=translations,
+        geojson_file='https://geoportal1-ons.opendata.arcgis.com/datasets/4fcca2a47fed4bfaa1793015a18537ac_4.geojson',
```

Comment:
Typically not a good idea to have a remote source for data in your tests. It would potentially break if there's a problem with your internet connection. Could you download this locally and test that instead?

Reply:
Good point, I've downloaded and committed the geojson to the repo.

Reply:
As a rule of thumb, unit tests should be fast, predictable and reliable. They should only fail if there's a bug in your code. If your code typically fetches data from a remote source, a good way to test this would be the usage of "mocks" or "test doubles" (in PyTest, it's called [monkeypatch](https://docs.pytest.org/en/latest/monkeypatch.html))